data/hai/hai reveal 4 middle.txt

```diff
@@ -1233,29 +1233,77 @@ mission "Hai Reveal [A11] Sound the Alarm"
-			`Teeneep places a gentle hand on Sayari's arm. "Better this than war."`
-			label alarm
-			`	Sayari's eyes are alive with thinly-veiled panic as she slams her hand down on the table. "Captain <last>, you must leave immediately. Get on your ship and fly to <planet> right now! The winds of change are afoot and we will have an election. They'll hit the threshold in maybe only twenty minutes, and it'll take maybe thirty more for the bureaucracy to act on it. The instant the election is announced formally, all non-essential ships will be grounded, all communication in or out of Hai space will be cut off, and the border will be sealed to everyone - not in the polite way either. If even one ship tries to cross the border to investigate... there's no telling what will happen."`
+			`Upon your return, Teeneep sends an urgent demand for an audience with the Elders, along with a few paragraphs written in Hai. The meeting is called within an hour, and the room is entirely full. This may be the entire Elder Council; if so, they must consider this a grave matter. Teeneep and Sayari are waiting for you to follow them into the chambers.`
+			`	The Elder moderating the proceedings calls Teeneep to testify, using your language. It seems they consider your participation critical.`
+			`	Teeneep begins. "For the first time in three hundred years, we find ourselves beset on our borders by hostile forces. They have kidnapped five thousand of our people, and this council has been too paralyzed with fear of change to defend us. The Unfettered were once the defenders of all Hai, and I have convinced them to be so once again. They are ready to send a fleet to human space and extinguish the pirate threat. Will you do what is right for our people and negotiate this with the humans, or will you be replaced?"`
+			`	At the word "replaced," murmurs rise, and Sayari looks at you with deep concern. The moderator silences the room and asks you to testify.`
+			`	"Captain <last>, can you speak to the truth of these disappearances?"`
+			choice
+				`	"The evidence that there are ongoing kidnappings is circumstantial, but compelling. We must expand the investigation."`
+					goto uncertainties
+				`	"It is a coordinated kidnapping campaign, and we must put a stop to it but take care not to start a war."`
+			`	You hear many murmurs of dissent, but the moderator raises his paws to delay discussion.`
+				goto "next question"
+			label uncertainties
+			`	Teeneep looks mildly perturbed, while many Elders nod with agreement.`
+			label "next question"
+			`	The moderator turns back to you with a highly serious face. "Captain <last>, you have the unique vantage point of an accomplished outsider who has seen all parts of this conflict. We have become unwittingly embroiled in human internal conflicts and have been attacked for the first time in 300 years. Much of this is beyond our experience, but perhaps not beyond yours. What do you advise?"`
```

Comment:
Again, the 300 years.
However, in this context it is worth considering that the player is not a politician, their character has no diplomatic or policy backgrounding.
If some subset of the Elders are actually engaged enough to want to address possible options here, it seems to me far more likely that they would speak of a selection of options that they've already been given or requested from various people (or have devised themselves) and query you on which one you think is most reasonable/problematic. (As an outsider you can't be expected to propose a solution, but *can* be expected to spot a problem that others might have overlooked.)

I would expect that the options they'd present would looks something like:
- Their favourite
- The one they think is most practical
- The one they think is most ridiculous

Not necessarily in that order, and not necessarily with any indication of which is which.